src/Integration.Vsix/SonarLintTracker/IssuesSnapshot.cs

```diff
@@ -0,0 +1,116 @@
+            this.versionNumber = versionNumber;
+        }
+
+        public override int Count
+        {
+            get
+            {
+                return this.IssueMarkers.Count;
+            }
+        }
+
+        public override int VersionNumber
+        {
+            get
+            {
+                return versionNumber;
+            }
+        }
+
+        public override bool TryGetValue(int index, string columnName, out object content)
+        {
+            if ((index >= 0) && (index < this.IssueMarkers.Count))
+            {
+                // TODO missing issue type, such as bug, vulnerability, code smell
+                if (columnName == StandardTableKeyNames.DocumentName)
+                {
+                    content = filePath;
+                    return true;
+                }
+                else if (columnName == StandardTableKeyNames.Line)
+                {
+                    content = this.IssueMarkers[index].Span.Start.GetContainingLine().LineNumber;
+                    return true;
+                }
+                else if (columnName == StandardTableKeyNames.Column)
+                {
+                    var position = this.IssueMarkers[index].Span.Start;
+                    var line = position.GetContainingLine();
+                    content = position.Position - line.Start.Position;
+                    return true;
+                }
+                else if (columnName == StandardTableKeyNames.Text)
+                {
+                    content = this.IssueMarkers[index].Issue.Message;
+                    return true;
+                }
+                else if (columnName == StandardTableKeyNames.ErrorSeverity)
+                {
+                    content = __VSERRORCATEGORY.EC_WARNING;
+                    return true;
+                }
+                else if (columnName == StandardTableKeyNames.BuildTool)
+                {
+                    // TODO get correct analyzer name
+                    content = "SonarJS [SonarLint for Visual Studio 2015]";
+                    return true;
+                }
+                else if (columnName == StandardTableKeyNames.ErrorCode)
+                {
+                    content = this.IssueMarkers[index].Issue.RuleKey;
+                    return true;
+                }
+                else if ((columnName == StandardTableKeyNames.ErrorCodeToolTip) || (columnName == StandardTableKeyNames.HelpLink))
+                {
+                    // TODO use correct version
+                    // TODO add JavaScript rules on website
+                    content = string.Format(CultureInfo.InvariantCulture, "http://www.sonarlint.org/visualstudio/rules/index.html#version=5.9.0.992&ruleId={0}", this.IssueMarkers[index].Issue.RuleKey);
+                    return true;
+                }
+                else if (columnName == StandardTableKeyNames.ProjectName)
+                {
+                    // TODO get project name
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)